research/2d-data.py

```diff
@@ -0,0 +1,51 @@
+import importlib
+import os
+from types import GeneratorType
+
+print("What do you want to do?")
+print("[1] Encode a message")
+print("[2] Decode a message")
+try:
+    choice = int(input("Enter your choice: "))
+except ValueError:
+    print("Choose a number pls")
+    exit(3)
+
+# Encode a message
+if choice == 1:
+    data_size = int(os.getenv("QR_BITS_PER_CHUNK", 5))
+
+    sqr_printer = importlib.import_module("2d-decoding.square_printer")
+
+    msg = input("Enter an integer: ")
+    try:
+        n = int(msg)
+    except ValueError:
+        print("Not an integer lol")
+        exit(1)
+
+    from encoding_2d_data import encode
```

Comment:
Put imports at the top of files: [pep8](https://peps.python.org/pep-0008/#:~:text=Imports,-Imports%20should%20usually&text=Imports%20are%20always%20put%20at,Standard%20library%20imports.)